lazy_tensor_core/lazy_tensor_core/csrc/aten_ltc_bridge.cpp

```diff
@@ -247,16 +247,9 @@ c10::optional<Device> GetLtcDevice(const c10::optional<c10::Device>& device) {
 
 Device AtenDeviceToLtcDevice(const c10::Device& device) {
   CHECK_EQ(device.type(), at::kLazy) << device;
-  int ordinal = device.has_index() ? device.index() : -1;
-  if (ordinal < 0) {
-    c10::Device current_device = GetCurrentAtenDevice();
-    if (current_device.has_index()) {
-      ordinal = current_device.index();
-    }
-  }
-  if (ordinal < 0) {
-    return GetCurrentDevice();
-  }
+  // Ordinal doesn't make any sense now given
```

Comment:
it's not strictly true, sometimes people doing single-gpu work on a shared machine still want to set ordinal to a particular gpu to avoid conflict.  I don't know if that even applies to lazy:{ordinal} though, or if we override the actual GPU ordinal inside the backend.  That's something we can fix later, but I just wanted you to be aware.

Reply:
Since [TSBackendImpl::GetLocalDevices()](https://github.com/pytorch/pytorch/blob/lazy_tensor_staging/lazy_tensor_core/lazy_tensor_core/csrc/ts_backend/backend_impl.cpp#L178) only support one element, I don't think we can set "lazy:1" currently. Also see: [link](https://github.com/pytorch/pytorch/blob/lazy_tensor_staging/lazy_tensor_core/lazy_tensor_core/csrc/aten_ltc_bridge.cpp#L34). Let me rephrase the comments.

Reply:
ok; yea this part we can fix later

Reply:
This means we hardcode the `lazy:0` to be the default right? I felt like it is better to call `GetDefaultDevice` here instead. 

In the case of pt/lxa, `xla;0` is usually a `cpu` and `xla:1` is a `tpu`. We don't want to return `xla:0` as the default.

Reply:
Okay, then it looks like we need a way to actually query the default device in the backend interface. W.R.T this particular code, it's going to remove by Will in #67869 anyway. So it shouldn't matter.

Reply:
interesting. so when users use xla for single node tpu today, they have to move their tensors to xla:1?  

(or, they move to `xla` and the default device ordinal is 1?)

Reply:
We don't ask user to manually set device name like `xla:1`. We have a API [xm.xla_device()](https://github.com/pytorch/xla/blob/f9eb88c7216332dd11600e58ffd3067622c263fd/torch_xla/core/xla_model.py#L218) which will also set the [defualt_device](https://github.com/pytorch/xla/blob/f9eb88c7216332dd11600e58ffd3067622c263fd/torch_xla/core/xla_model.py#L240).